projects/fsastorefrontlib/src/cms-components/b2b/user/create/unit-user-create.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import {
+  UnitUserCreateComponent,
+  UserItemService,
+} from '@spartacus/organization/administration/components';
+import { ɵi as UnitUserItemService } from '@spartacus/organization/administration/components';
```

Comment:
Maybe put some comment above this import so it's easier to troubleshoot, because it's highly probable this feature will break once we upgrade Spartacus version.